src/views/domain-page/domain-page-settings/domain-page-settings.styles.ts

```diff
@@ -1,7 +1,18 @@
 import { styled as createStyled, type Theme } from 'baseui';
+import { type ToasterOverrides } from 'baseui/toast';
 
 export const styled = {
   SettingsContainer: createStyled('div', ({ $theme }: { $theme: Theme }) => ({
     paddingTop: $theme.sizing.scale950,
   })),
 };
+
+export const overrides = {
+  toast: {
+    ToastBody: {
+      style: {
+        width: '500px',
```

Comment:
Can you test this on different screen-sizes

Reply:
Good call, I've put this behind a mediaquery.